static/js/containers/Drawer_test.js

```diff
@@ -10,99 +10,173 @@ import { ResponsiveDrawer, mapStateToProps } from "./Drawer"
 import { setShowDrawerMobile } from "../actions/ui"
 import { DRAWER_BREAKPOINT } from "../lib/util"
 import * as selectors from "../lib/redux_selectors"
-import * as utilFuncs from "../lib/util"
+import * as util from "../lib/util"
+import * as channelLib from "../lib/channels"
+import { channelURL, newPostURL } from "../lib/url"
 import { makeChannelList } from "../factories/channels"
+import { shouldIf } from "../lib/test_utils"
 
-describe("Drawer tests", () => {
-  let sandbox
+describe("Drawer", () => {
+  let sandbox, dispatchStub, channels, isMobileWidthStub, getViewportWidthStub
 
   beforeEach(() => {
     sandbox = sinon.createSandbox()
+    dispatchStub = sandbox.stub()
+    isMobileWidthStub = sandbox.stub(util, "isMobileWidth").returns(true)
+    getViewportWidthStub = sandbox.stub(util, "getViewportWidth")
+    channels = makeChannelList()
   })
 
   afterEach(() => {
     sandbox.restore()
   })
 
-  describe("drawer component", () => {
-    let dispatchStub, channels, isMobileWidthStub, getViewportWidthStub
+  const renderDrawer = (props = {}) =>
```

Comment:
I removed this describe block and moved the tests within it to the top level describe block. There didn't seem to be a good reason to have these tests and the setup logic in an inner describe block